1/2_add_transport.md

```diff
@@ -1,9 +1,53 @@
+
+So without any further ado, lets go ahead and start discussing and implementing the protocols that we'll be using.
+
+### Various Transport Protocols, we'll be using
+
+* `libp2p-tcp` : This is the libp2p implementation of the standard TCP (Transfer Control Protocol). 
+
+	*The basic function of  **TCP** is to segment the information into smaller data packets, index them, and add additional information such as source & destination.* 
+
+	More information about TCP is given in the the [first section of this workshop](http://salabs.me/jslibp2p-chat-tutorial/#/1/introduction). It could be simply installed using the command:
+	```bash
+	npm i libp2p-tcp@0.13.0
+	```
+>Note: For this workshop, we'll be using specific versions of the transport libraries; so make sure you only install the above mentioned protocol. 
+
+* `libp2p-websockets` : This is the libp2p implementation of a transport protocol called **Websockets**. Websockets is an independent protocol built over TCP. 
+
+	One major advantage of **Websockets** is that it *enables a rich and efficient duplex (two-way) connection enabling peers to flow information both the ways simultaneously with a high speed*. For learning more about the basic websockets structure, checkout this blog [here](https://www.tutorialspoint.com/websockets/websockets_quick_guide.htm).
+
+	
+	It could be simply installed using the command:
+	```bash
+	npm i libp2p-websockets@0.12.2
+	```
+* `libp2p-webrtc-star` : Libp2p-webrtc-star is one of the open-source WebRTC transport protocol available to Libp2p.  
+
+	For starters, [WebRTC](https://webrtc.org/) enables browsers and mobile applications to establish Real-Time Communications (RTC) capabilities via simple APIs. `libp2p-webrtc-star` includes the WebRTC transport protocol along with a discovery service. For learning more about it's functionalities and implementations, checkout the [its official repository](https://github.com/libp2p/js-libp2p-webrtc-star).
+
+* `wrtc` : **wrtc** *isn't a libp2p transport library* but rather a **node.js** implementation of **WebRTC** (discussed above). It's shipped along with a few additional helpful, non-standard extensions. Checkout [the official repository of node-wrtc](https://github.com/node-webrtc/node-webrtc).
+
+	It could be simply installed using the command:
+	```bash
+	npm i wrtc@0.4.1
+	```
+* `libp2p-mdns` : Libp2p-mdns is the libp2p implementation of Multi-cast DNS. For starters, **Multi-cast DNS** *is a way of using familiar DNS programming interfaces, packet formats and operating semantics, in a small network where no conventional DNS server has been installed.* For learning more libp2p-mdns, checkout [its official repository](https://github.com/node-webrtc/node-webrtc).
```

Comment:
This is too technical can we write something better?

Reply:
Sure I'll start working on it!

Reply:
Shresth do you want me to just refactor the literature for Multi-cast DNS or for all other protocols as well?

Reply:
rather than putting the technical definition put what it does, from what I remember what make mdns cool is it was made by apple to connect to printers and wireless devices without any configuration. Also all the apple magic when you can share your wifi password just by bringing your iphone close to another is also mdns. read a bit more about the it and write some more cool context!